### PR TITLE
Booking days off

### PR DESCRIPTION
**Pull request by taniafecheta**

### CODE REVIEW

File: doc/BookingDaysOff.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Booking days off
+
+Returns a JSON with days off from an user
```
Comment (h-mihail) [inline doc/BookingDaysOff.md]: Creates a new entry*

File: logs/errors.log [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+[2018-05-31T10:25:15+03:00] GET "/users/-1" 404 Not Found - {}
+[2018-05-31T10:25:15+03:00] GET "/users/A" 404 Not Found - {}
+[2018-05-31T10:25:15+03:00] GET "/users/%220%22" 404 Not Found - {}
+[2018-05-31T10:27:02+03:00] DELETE "/days/-1" 404 Not Found - {}
+[2018-05-31T10:27:02+03:00] DELETE "/days/A" 404 Not Found - {}
+[2018-05-31T10:27:02+03:00] DELETE "/users/-1" 404 Not Found - {}
+[2018-05-31T10:27:02+03:00] DELETE "/users/A" 404 Not Found - {}
+[2018-05-31T10:27:02+03:00] GET "/users/-1" 404 Not Found - {}
+[2018-05-31T10:27:02+03:00] GET "/users/A" 404 Not Found - {}
+[2018-05-31T10:27:02+03:00] GET "/users/%220%22" 404 Not Found - {}
```
Comment (h-mihail) [inline logs/errors.log]: This should be added to ```.gitignore```

File: src/actions/bookingDaysOffActions.js [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+}
+
+const createDaysOffArray = (startDate, endDate) => {
+    return new Promise((resolve, reject) => {
+        startDate = new Date(startDate);
+        endDate = new Date(endDate);
+
+        let daysOffArray = [];
+        let month = "";
+        let day = "";
```
Comment (taniafecheta) [inline src/actions/bookingDaysOffActions.js]: nu mai sunt folosite month si day

File: src/actions/bookingDaysOffActions.js [hunk 4]
```diff
@@ -0,0 +1,160 @@
+const fs = require("fs");
```
Comment (h-mihail) [inline src/actions/bookingDaysOffActions.js]: fs module can be removed now

File: src/actions/bookingDaysOffActions.js [hunk 5]
```diff
@@ -7,7 +7,13 @@ const {
 
 const isDateValid = date => {
     const regex = /([0-9]{4})(-)([1][0-2]|[0][1-9])(-)([0][1-9]|[1-2][0-9]|([3][0-1]))/;
-    return date !== undefined && typeof date === "string" && regex.test(date) && (new Date(date).getFullYear()) <= (new Date().getFullYear() + 3);
+    return date !== undefined && typeof date === "string" && regex.test(date) ;
```
Comment (andreicioromila) [inline src/actions/bookingDaysOffActions.js]: Do you need all these conditions?

**Review by andreicioromila — CHANGES_REQUESTED** [review]

**Review by h-mihail — DISMISSED** [review]

**Review by h-mihail — COMMENTED** [review]
.

**Review by climandrei94 — DISMISSED** [review]

**Review by andreicioromila — DISMISSED** [review]

**Review by climandrei94 — DISMISSED** [review]

**Review by ghost — APPROVED** [review]

**Review by climandrei94 — APPROVED** [review]

**Review by h-mihail — APPROVED** [review]

**Pull request merged**